R/froll.R

```diff
@@ -111,6 +111,47 @@ make.roll.names = function(x.len, n.len, n, x.nm, n.nm, fun, adaptive) {
   ans
 }
 
+# irregularly spaced time series, helper for creating adaptive window size
+frolladapt = function(x, n, align="right", partial=FALSE, give.names=FALSE) {
+  x = unclass(x)
+  if (!is.numeric(x))
+    stopf("'x' must be of a numeric type")
+  if (!is.integer(x))
+    x = as.integer(x)
```

Comment:
Is there a TODO to support double storage `x`? What's the technical limitation here?